pkg/acsengine/engine.go

```diff
@@ -454,6 +454,11 @@ func getParameters(cs *api.ContainerService, isClassicMode bool, generatorCode s
 	addValue(parametersMap, "fqdnEndpointSuffix", cloudSpecConfig.EndpointConfig.ResourceManagerVMDNSSuffix)
 	addValue(parametersMap, "targetEnvironment", GetCloudTargetEnv(location))
 	addValue(parametersMap, "linuxAdminUsername", properties.LinuxProfile.AdminUsername)
+	if properties.LinuxProfile.CustomSearchDomain != nil {
+		addValue(parametersMap, "sdName", properties.LinuxProfile.CustomSearchDomain.Name)
```

Comment:
Can we make the template param a little more descriptive? E.g., `customSearchDomainName` instead of `sdName`?

Reply:
Yes, I agree. And about the CLA signing I am waiting for legal.